### PR TITLE
chore: remove react-keytips scaffold

### PR DESCRIPTION
**Pull request by mainframev**
Removes previously generated scaffold package for react-keytips, which is stored inside the fluentui-contrib repo.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -374,8 +374,6 @@ packages/react-components/react-utilities-compat/library @microsoft/cxe-prg
 packages/react-components/react-utilities-compat/stories @microsoft/cxe-prg
```
Comment (fabricteam) [inline .github/CODEOWNERS:374]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33176&commitId=4dab274efc011d388fefecda2b6463b782bfdd0c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUIV0 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

File: .github/CODEOWNERS (hunk 1 above)
Comment (fabricteam) [inline .github/CODEOWNERS:374]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33176&commitId=4dab274efc011d388fefecda2b6463b782bfdd0c&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: .github/CODEOWNERS (hunk 1 above)
Comment (fabricteam) [inline .github/CODEOWNERS:374]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33176&commitId=4dab274efc011d388fefecda2b6463b782bfdd0c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|50| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|5| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by tudorpopams — APPROVED** [review]

**Pull request merged**